src/Features/Auth/ForgetPassword/index.js

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useEffect } from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory } from 'react-router';
+
+import FormComponent from '../../../components/FormComponent';
+import { emailRegex } from '../../../scripts/constants';
+import { forgotPassword } from '../actions';
+
+function ForgetPassword() {
+  const history = useHistory();
+  const dispatch = useDispatch();
+  const { message, status } = useSelector((state) => {
+    const { message, status } = state.responseMessage;
+    return { message, status };
+  });
+  const forgetPasswordClickHandler = (e) => {
+    e.preventDefault();
+
+    let isValid = true;
```

Comment:
Isn't it good that we make a seprate separate function validateOnSubmit and then just call that here .. ?